packages/convergence/src/convergence.js

```diff
@@ -92,21 +92,31 @@ class Convergence {
   /**
    * Creates a new convergence with the given assertion added to its
    * stack. The new convergence's initial stack will be inherited from
-   * this convergence instance. The assertion given to `.once()` will
+   * this convergence instance. The assertion given to `.when()` will
    * be converged on using the `convergeOn` helper, but it's timeout
    * will be managed by this convergence instance
    *
    * @param {Function} assert - the assertion to converge on
    * @returns {Convergence} a new convergence instance
    */
-  once(assert) {
+  when(assert) {
     return new this.constructor({
       _stack: [{ assert }]
     }, this);
   }
 
   /**
-   * Similar to `.once()`, creates a new convergence with the given
+   * Alias for `.when()`
+   *
+   * @deprecated
+   * @returns {Convergence} a new convergence instance
+   */
+  once() {
+    return this.when(...arguments);
```

Comment:
Can we add a `console.warn` here so if anyone is using it they'll know to migrate?

Reply:
> I didn't add a deprecation warning message to .once() since this is pre-v1 anyway.

Plus, if we log a warning, test runners will get slow when logging a thousand of them. I think not warning is okay since this is all alpha / beta currently

Reply:
I think if you're an early adopter you're going to want to be made aware of the churn ASAP, because early adopters are more tolerant of churn in order to get the hawt features. It'll make their lives easier. And, after all, you're _upgrading_ the version of the framework. 

Maybe having an easy way to silence the deprecations?

Reply:
What would be an easy way to silence it? It is just one warn, so I don't think we need to introduce a deprecation mechanism for it.